syft/pkg/cataloger/java/archive_parser.go

```diff
@@ -23,6 +23,10 @@ var archiveFormatGlobs = []string{
 	"**/*.ear",
 	"**/*.jpi",
 	"**/*.hpi",
+	"**/*.lpkg", // Zip-compressed package used to deploy applications
+	// (aka plugins) to Liferay users that can be installed on their
+	// websites, contains .JAR files and a .PROPERTIES file, the latter
+	// has information about the application and installation requirements.
```

Comment:
What do you think it would look like to add a test for this?

Reply:
I added a file test. Their docs and tools are pretty outdated and I can't find sample projects, even on GitHub.